lab-sarah/test/recipe-route-test.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+
+const request = require('superagent');
+const expect = require('chai').expect;
+
+require('../server.js');
+
+describe('testing recipe routes', function(){
+  var recipe = null;
+  describe('testing POST /api/recipe', function(){
+    it('should return a recipe', function(done){
+      request.post('localhost:3000/api/recipe')
+      .send({name: 'Cake', content: 'flour and chocolate'})
+      .end((err, res) => {
+        if (err) return done(err);
+        expect(res.status).to.equal(200);
+        console.log('body',res.body);
+        expect(res.body.name).to.equal('Cake');
+        expect(res.body.content).to.equal('flour and chocolate');
+        recipe = res.body;
+        done();
+      });
+    });
+    it('should return a response status of 400', function(done){
```

Comment:
Try to be more explicit in your descriptions.... **when** should it return a response status of 400?